### PR TITLE
Added ROS2 introduction

### PR DESCRIPTION
**Pull request by jsantiago-eProsima**
Signed-off-by: Javier Santiago <javiersantiago@eprosima.com>

### CODE REVIEW

File: docs/rst/introduction/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.. _vulcanexus_global_introduction:
+
+Vulcanexus Overview
+===================
+
+*Vulcanexus*, the *ROS 2* all-in-one tool set, expands the *ROS 2* environment improving the developer experience with *ROS 2* providing open source features and tools not available with the current `ROS 2 release <https://www.ros.org/reps/rep-2000.html#release-schedule>`_.
+Consequently, *Vulcanexus* shares the underlying concepts, principles and architecture of :doc:`ROS 2 <../../ros2_documentation/source/index>`.
+For more information about *ROS 2* please refer to the 2022 Science Robotics vol. 7 (May 2022) paper `Robot Operating System 2: Design, architecture, and uses in the wild <https://www.science.org/doi/10.1126/scirobotics.abm6074>`_ by S. Macenski, T. Foote, B. Gerkey, C. Lalancette and W. Woodall.
```
Comment (rsanchez15) [inline docs/rst/introduction/introduction.rst:8]: I'd include this as a footnote. Check [here](https://devopstutodoc.readthedocs.io/en/stable/documentation/doc_generators/sphinx/rest_sphinx/footnotes.html) how to do so

File: docs/rst/introduction/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Another category includes robotic application's algorithms.
+*Vulcanexus*, being a *ROS 2* all-in-one tool set, provides the algorithms included in *ROS 2* ecosystem.
+
+Developer tools
+^^^^^^^^^^^^^^^
+
+Finally, *ROS 2* provides several development tools for debugging, logging, visualization, introspection, simulation, etc.
+*Vulcanexus* adds to *ROS 2* toolset including several tools:
+
+- `eProsima Fast DDS Monitor <https://fast-dds-monitor.readthedocs.io/en/latest/>`_: graphical desktop application to monitor DDS communications when using *eProsima Fast DDS*.
```
Comment (rsanchez15) [inline docs/rst/introduction/introduction.rst:46]:
```suggestion
- `ROS 2 Monitor <https://fast-dds-monitor.readthedocs.io/en/latest/>`_: graphical desktop application to monitor ROS 2 communications.
```

File: docs/rst/introduction/introduction.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- `eProsima Shapes Demo <https://eprosima-shapes-demo.readthedocs.io/en/latest/index.html>`_: first demo application to understand the most used DDS Quality of Service (QoS) and test DDS and *ROS 2* communication.
```
Comment (rsanchez15) [inline docs/rst/introduction/introduction.rst:47]:
```suggestion
- `ROS 2 Shapes Demo <https://eprosima-shapes-demo.readthedocs.io/en/latest/index.html>`_: first demo application to understand the most used ROS 2 Quality of Service (QoS) and test DDS and *ROS 2* communication.
```

File: docs/rst/introduction/introduction.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- :ref:`micro-ROS <tutorials_micro_introduction_microros>`: default embedded toolkit for *ROS 2* and *Vulcanexus*.
+- `eProsima DDS Router <https://eprosima-dds-router.readthedocs.io/en/latest/index.html>`_: end-user software application enabling the connection of distributed DDS networks.
```
Comment (rsanchez15) [inline docs/rst/introduction/introduction.rst:49]:
```suggestion
- `ROS 2 Router <https://eprosima-dds-router.readthedocs.io/en/latest/index.html>`_: end-user software application enabling the connection of distributed ROS 2 environments.
```

File: docs/rst/introduction/metapackages.rst [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+===============================
+
+*Vulcanexus* provides several binary distributions that install specific packages depending on the developer's needs.
+:ref:`vulcanexus_introduction` table summarizes the different *Vulcanexus* package distributions along with the most important included packages.
+
+Vulcanexus Core
+---------------
+
+**Vulcanexus Core** is the most fundamental and minimal *Vulcanexus* installation.
+It is composed by the `ROS 2 Base libraries <https://ros.org/reps/rep-2001.html#end-user-entry-points>`_ but with the latest `eProsima Fast DDS <https://fast-dds.docs.eprosima.com/en/latest/>`_ and `Fast CDR <https://www.eprosima.com/images/PDFs/Fast_CDR.pdf>`_ releases and the RMW needed to benefit from the latest features included in those releases.
```
Comment (rsanchez15) [inline docs/rst/introduction/metapackages.rst:13]:
```suggestion
It is composed by the `ROS 2 Base libraries <https://ros.org/reps/rep-2001.html#end-user-entry-points>`_ but with the latest `eProsima Fast DDS <https://fast-dds.docs.eprosima.com/en/latest/>`_ and `Fast CDR <https://www.eprosima.com/images/PDFs/Fast_CDR.pdf>`_ releases and the ROS MiddleWare (RMW) needed to benefit from the latest features included in those releases.
```

File: docs/rst/introduction/metapackages.rst [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+        .. code-block:: bash
+
+            sudo apt install vulcanexus-humble-core
+
+Vulcanexus Tools
+----------------
+
+**Vulcanexus Tools** is focused on helping developers with the introspection and debugging of their system.
+Thus, the `eProsima Fast DDS Monitor <https://fast-dds-monitor.readthedocs.io/en/latest/>`_ and its dependency (`eProsima Fast DDS Statistics Backend <https://fast-dds-statistics-backend.readthedocs.io/en/latest/>`_) are included in order to monitor and diagnose the DDS communication network.
```
Comment (rsanchez15) [inline docs/rst/introduction/metapackages.rst:34]:
```suggestion
Thus, the `ROS 2 Monitor <https://fast-dds-monitor.readthedocs.io/en/latest/>`_ and the `ROS 2 Statistics Backend <https://fast-dds-statistics-backend.readthedocs.io/en/latest/>`_ are included in order to monitor and diagnose the ROS 2 communication network.
```

File: docs/rst/introduction/metapackages.rst [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+Also, `eProsima Shapes Demo <https://eprosima-shapes-demo.readthedocs.io/en/latest/index.html>`_ is provided as a simple demo to help testing that the communication is successfully established in the system.
```
Comment (rsanchez15) [inline docs/rst/introduction/metapackages.rst:35]:
```suggestion
Also, `ROS 2 Shapes Demo <https://eprosima-shapes-demo.readthedocs.io/en/latest/index.html>`_ is provided as a simple demo to help testing that the communication is successfully established in the system.
```

File: docs/rst/introduction/metapackages.rst [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+        .. code-block:: bash
+
+            sudo apt install vulcanexus-humble-micro
+
+Vulcanexus Cloud
+----------------
+
+**Vulcanexus Cloud** is oriented to geographically spaced scenarios, deploying robotic applications through a distributed network.
+To this end, `eProsima DDS Router <https://eprosima-dds-router.readthedocs.io/en/latest/index.html>`_ is installed in this *Vulcanexus* package distribution, allowing DDS communication through different network layers (bridging the local networks through WAN communication).
```
Comment (rsanchez15) [inline docs/rst/introduction/metapackages.rst:76]:
```suggestion
To this end, `ROS 2 DDS Router <https://eprosima-dds-router.readthedocs.io/en/latest/index.html>`_ is installed in this *Vulcanexus* package distribution, allowing ROS 2 communications through different network layers (bridging the local networks through WAN communication).
```

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: docs/rst/introduction/introduction.rst [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+Vulcanexus Overview
+===================
+
+*Vulcanexus*, the *ROS 2* all-in-one tool set, expands the *ROS 2* environment improving the developer experience with *ROS 2* providing open source features and tools not available with the current `ROS 2 release <https://www.ros.org/reps/rep-2000.html#release-schedule>`_.
+Consequently, *Vulcanexus* shares the underlying concepts, principles and architecture of :doc:`ROS 2 <../../ros2_documentation/source/index>`.
+For more information about *ROS 2* please refer to the `Robot Operating System 2: Design, architecture, and uses in the wild <https://www.science.org/doi/10.1126/scirobotics.abm6074>`_ [#]_.
+This section will follow and summarize the most important points contained in this paper, pointing out the differences and improvements that *Vulcanexus* provides.
+
+.. [#] S. Macenski, T. Foote, B. Gerkey, C. Lalancette and W. Woodall, "Robot Operating System 2: Design, architecture, and uses in the wild", Science Robotics vol. 7, May 2022
```
Comment (rsanchez15) [inline docs/rst/introduction/introduction.rst:11]: I'd move this to the end of the page (footnote) as this page is short enough

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: docs/rst/introduction/introduction.rst (hunk 9 above)
Comment (JLBuenoLopez-eProsima) [reply docs/rst/introduction/introduction.rst]: Done in 0d61e2fc8bfb6587e04c98238b18414c92b2c0b2

**Review by rsanchez15 — APPROVED** [review]

**Pull request merged**